lib/facets/create-documents-facets.js

```diff
@@ -0,0 +1,15 @@
+module.exports = function createDocumentFacets (results) {
+  var archiveValues = [];
+  results.aggregations.documents.archive.archive_filters.buckets.forEach(value => {
```

Comment:
is `results.aggregations.documents.archive.archive_filters.buckets` _always_ set...?
if so you could use a `.map` instead of `forEach` and assign the array directly.